easybuild/easyconfigs/a/almosthere/almosthere-1.0.1-GCCcore-7.3.0.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'CMakeMake'
+
+name = 'almosthere'
+version = '1.0.1'
+
+homepage = 'https://github.com/horta/almosthere'
+description = """Progress indicator C library.
+
+ATHR is a simple yet powerful progress indicator library that works on Windows, 
+Linux, and macOS. It is non-blocking as the progress update is done via a 
+dedicated, lightweight thread, as to not impair the performance of the calling program."""
+
+toolchain = {'name': 'GCCcore', 'version': '7.3.0'}
+toolchainopts = {'pic': True}
+
+github_account = 'horta'
+source_urls = [GITHUB_SOURCE]
+sources = ['%(version)s.tar.gz']
+checksums = ['f9bc5d76370548dcdc5383063fd1896b35c5f64433166f5ea67708bfeb69c832']
+
+builddependencies = [
+    ('binutils', '2.30'),
+    ('CMake', '3.12.1'),
+]
+
+configopts = '-DBUILD_SHARED_LIBS=ON'
```

Comment:
Please also make this

```python
configopts = ['-DBUILD_SHARED_LIBS=ON', '-DBUILD_SHARED_LIBS=OFF']
```

Reply:
It's not necessary in this version, both the static and shared libs are created in one go with `-DBUILD_SHARED_LIBS=ON`.